src/rebar_dir.erl

```diff
@@ -35,6 +35,7 @@ base_dir(State) ->
 profile_dir(State, Profiles) ->
     {BaseDir, ProfilesStrings} = case [ec_cnv:to_list(P) || P <- Profiles] of
         ["global" | _] -> {?MODULE:global_cache_dir(State), [""]};
+        ["bootstrap", "default"] -> {rebar_state:get(State, base_dir, ?DEFAULT_BASE_DIR), ["default"]};
```

Comment:
Makes sense then. I'd probably just add a comment that this is strictly for bootstrapping rebar3 in the right directory.